backend/controllers/authController.js

```diff
@@ -1,18 +1,16 @@
-const db = require("../models"); // Ensure models are required correctly
+const db = require("../models"); // Make sure to require your models
 const jwt = require("jsonwebtoken");
 const bcrypt = require("bcrypt");
 
-// Token generation helper function
-const generateToken = (payload, secret, expiresIn) => {
-   return jwt.sign(
-      payload,
-      secret,
-      { expiresIn }
-   );
-}
-
-// Verify Token contoller - refresh if (access) token is invalid
-const verifyToken = async (req, res, next) => {
+// generate token function from .env settings
+const generateToken = (payload) => {
+  return jwt.sign(payload, process.env.JWT_SECRET, {
+    expiresIn: process.env.JWT_EXPIRES_IN,
+  });
+};
+
```

Comment:
I would prefer to keep jwt, and bcrypt implementations out of the authController, in a place like /lib

Reply:
I've put it in the helpers folder